### PR TITLE
showcase long-running shell commands

### PR DESCRIPTION
**Pull request by mingyuan-xia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -787,6 +788,41 @@ client.listDevices()
...
+var adb = require('adbkit')
+var client = adb.createClient()
+
+client.listDevices()
+  .then(function(devices) {
+    return Promise.map(devices, function(device) {
+      return client.shell(device.id, 'logcat') // logcat just for illustration,
+                                               // prefer client.openLogcat in real use 
+        .then(function(conn) {
+          var lines = 0
```
Comment (NoahAndrews) [inline README.md:803]: The rest of the example uses "line".

File: README.md [hunk 2]
```diff
@@ -787,6 +788,41 @@ client.listDevices()
...
+* Progressively read the output of a long-running command and terminate it
+```js
+var Promise = require('bluebird')
+var adb = require('adbkit')
+var client = adb.createClient()
+
+client.listDevices()
+  .then(function(devices) {
+    return Promise.map(devices, function(device) {
+      return client.shell(device.id, 'logcat') // logcat just for illustration,
```
Comment (koral--) [inline README.md:800]: Why not use `client.openLogcat` then in example?

Comment (mingyuan-xia) [reply README.md:800]: just a showcase, need a command that never terminates and is familiar to everyone

File: README.md (hunk 1 above)
Comment (mingyuan-xia) [reply README.md]: thx, that has been fixed

**Review by koral-- — APPROVED** [review]

**Pull request merged**